package.json

```diff
@@ -57,9 +57,11 @@
   "dependencies": {
     "@polymer/polymer": "^3.0.0",
     "@vaadin/vaadin-avatar": "^1.0.3",
+    "@vaadin/vaadin-button": "^2.4.0",
     "@vaadin/vaadin-element-mixin": "^2.4.1",
     "@vaadin/vaadin-lumo-styles": "^1.6.1",
     "@vaadin/vaadin-material-styles": "^1.3.2",
+    "@vaadin/vaadin-text-field": "^3.0.0",
```

Comment:
Please use `^2.8.0` if you want the component to be compatible with Vaadin 14.x

Reply:
Done.